### DISCUSSION TITLE
Typographical error in English version?

### DISCUSSION THREAD

**Discussion by idrista** [open-sauced/intro#179] [Q&A] [answered]
In the "Prerequisites" section, I believe I've spotted a typographical error. Is it intentional?
« starting> this course » 

https://github.com/open-sauced/intro/blob/4b63c8022206374b0aadd73d694ac7e9e53dabef/docs/intro-to-oss/intro.md?plain=1#L9

**Comment by adiati98** [comment, marked as answer]
Hey @idrista,

Sorry for just seeing this! That's definitely a typo. Good catch! ✨ 

I think someone is already working on it, but it still has no issue. So, we haven't reviewed it yet.
You are welcome to create an issue to report this! 😄
